stdlib/data/recipe.lua

```diff
@@ -423,4 +423,4 @@ function Recipe:replace_result(result_name, normal, expensive, main_product)
     return self
 end
 
-return Recipe
+return Recipe
```

Comment:
Why was the newline removed?  Makes it not a valid posix text file.

Reply:
Was working on some changes to add a results replacer but haven't finished so I attempted to remove the changes I've made as I only wanted to upload the tech changes to fix the add_pack not working